pkg/satellite/validation.go

```diff
@@ -0,0 +1,57 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package satellite
+
+import (
+	"regexp"
+	"strings"
+)
+
+// IsValid is a method for validating UserInfo entity
+func (userInfo *UserInfo) IsValid() bool {
+	if userInfo.FirstName == "" {
```

Comment:
Instead of return just "true" or "false", actually return what is wrong with the user info, it's more helpful when trying to understand what is disallowed.